plugins/analytics/public/packettotal.py

```diff
@@ -0,0 +1,147 @@
+import json
+import logging
+
+import requests
+
+from core.analytics import OneShotAnalytics
+from core.errors import ObservableValidationError
+from core.observables import Hostname, Ip, Hash, Url, Text
+from core.config.config import yeti_config
+
+class PacketTotalAPI(object):
+    """Base class for querying the PacketTotal API."""
+    settings = {
+        'packettotal_api_key': {
+            'name': 'PacketTotal API Key',
+            'description': 'API Key provided by packettotal.com'
+        }
+    }
+
+    API = 'https://api.packettotal.com/v1/'
+
+    @staticmethod
+    def search(observable, api_key):
+        #"""
+        #:param observable: The extended observable class
+        #:param api_key: The api key obtained from PacketTotal
+        #:return:  packettotal json response or None if error
+        #"""
+
+        params = {'query': observable.value}
+
+        try:
+            res = requests.get(
+                '{}search'.format(PacketTotalAPI.API),
+                headers={'x-api-key': api_key},
+                params=params,
+                verify=False,
```

Comment:
Why set this to False? Seems like not the best practice...

Reply:
This was due to a self signed cert issue for a while. I will remove this and do a updated pr.